core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -195,10 +201,41 @@ private static boolean needsNormalization(String hostnamePattern) {
     }
 
     @Nullable
-    static SslContext validateSslContext(@Nullable SslContext sslContext) {
+    static SslContext validateSslContext(@Nullable SslContext sslContext) throws SSLException {
```

Comment:
Because this only called from `VirtualHostBuilder`, how about moving this into the class and make this private?

Reply:
I guess we can remove `@Nullable` here?

Reply:
After moving, 
can we also make this method non-static private method?
Do we have any reason this method should be static?

Reply:
I think there's not so much difference if we add static or not because it's optimized anyway. However, we prefer the class level method rather than the instance level. Please check out [here](https://www.geeksforgeeks.org/static-methods-vs-instance-methods-java/). 😉

Reply:
Thank you, it helps me a lot to understand :+1: